package.json

```diff
@@ -3,6 +3,7 @@
   "homepage": "https://libdemsoftware.gitlab.io/ldimagemaker/",
   "dependencies": {
     "bulma": "^0.9.2",
+    "create-react-app": "^4.0.3",
```

Comment:
I don't think this should be needed, given react-scripts already exists, could you try backing out this change and the yarn.lock and seeing if it still runs?

Reply:
Bum, I thought I'd reverted that - I didn't make it it was yarn on its own.